### PR TITLE
Gcoin 1.2.1

### PR DESCRIPTION
**Pull request by lbyyy**
[1.2.1] - 2017-06-12

Changed
Rorder type of transction.
Prevent same pubkey in a alliance redeem script.

Fixed
Fix voting bug when turn on txindex.
Fix rpc signrawtransaction for license/miner relative tx.
Fix assignfixedaddress.
Fix CheckTxFeeAndColor.
Fix bug when transfering license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Change Log
 All notable changes to this project will be documented in this file.
+
+## [1.2.1] - 2017-06-12
+### Changed
+- Reorder type of transction.
```
Comment (hihiben) [inline CHANGELOG.md]: transaction

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+- Prevent same pubkey in a alliance redeem script.
+
+### Fixed
+- Fix voting bug when turn on txindex.
```
Comment (hihiben) [inline CHANGELOG.md]: 這個是什麼？

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+- Fix rpc signrawtransaction for license/miner relative tx.
```
Comment (hihiben) [inline CHANGELOG.md]: relative? related?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+- Fix assignfixedaddress.
+- Fix CheckTxFeeAndColor.
+- Fix bug when transfering license.
```
Comment (hihiben) [inline CHANGELOG.md]: transferring

File: gcoin-community.spec [hunk 5]
```diff
@@ -61,6 +61,15 @@ make check
 
 %changelog
 
+* Sat June 12 2017 Bo-Yu Lin <skzlbyyy@gmail.com> - 1.2.1
```
Comment (hihiben) [inline gcoin-community.spec:64]: 同上

File: CHANGELOG.md (hunk 2 above)
Comment (lbyyy) [reply CHANGELOG.md]: -txindex

**Comment by lbyyy** [comment]
solved

**Review by hihiben — APPROVED** [review]

**Pull request merged**